reference-implementation/lib/abstract-ops/readable-streams.js

```diff
@@ -1236,7 +1251,10 @@ function ReadableByteStreamControllerCommitPullIntoDescriptor(stream, pullIntoDe
 
   let done = false;
   if (stream._state === 'closed') {
-    assert(pullIntoDescriptor.bytesFilled === 0);
+    assert(pullIntoDescriptor.bytesFilled % pullIntoDescriptor.elementSize === 0);
+    if (!pullIntoDescriptor.readFully) {
```

Comment:
Maybe make this if (readFully) { assert1 } else { assert2 }.

Even though your first assert makes sense in both cases, it feels a bit redundant and less clear.